Sources/SourceControl/GitRepository.swift

```diff
@@ -207,12 +210,21 @@ public struct GitRepositoryProvider: RepositoryProvider, Cancellable {
 
     public func isValidDirectory(_ directory: Basics.AbsolutePath) throws -> Bool {
         let result = try self.git.run(["-C", directory.pathString, "rev-parse", "--git-dir"])
```

Comment:
@francescomikulis pointed out that this actually calls `rev-parse` rather than `config --get remote.origin.url`. That is, it's just checking whether this is a git directory at all (rather than checking that the remote matches). Ignore me!